google-api-go-generator/gen.go

```diff
@@ -1986,6 +1992,32 @@ func (meth *Method) generateCode() {
 		pn("}")
 	}
 
+	if meth.supportsMediaUpload() && meth.api.Name == "storage" {
+		comment := "WithRetry causes the library to retry the initial request of the upload" +
+			"(for resumable uploads) or the entire upload (for multipart uploads) if" +
+			"a transient error occurs. This is contingent on ChunkSize being > 0 (so" +
+			"that the input data may be buffered). The backoff argument will be used to" +
+			"determine exponential backoff timing, and the errorFunc is used to determine" +
+			"which errors are considered retryable. By default, exponetial backoff will be" +
+			"applied using gax defaults, and the following errors are retried:" +
+			"\n\n" +
+			"- HTTP responses with codes 429, 502, 503, and 504." +
```

Comment:
Could the error codes be pulled from gax  instead of string literals?

Reply:
Seeing as this is just a one-off comment and these values are not likely to change, I don't think it's work adding extra complexity to the templating in order to accomplish this.